plugins/telemetry/telemetry_impl.cpp

```diff
@@ -8,47 +8,28 @@
 
 namespace dronecore {
 
-TelemetryImpl::TelemetryImpl(System &system) :
-    PluginImplBase(system),
-    _position_mutex(),
-    _position(Telemetry::Position {double(NAN), double(NAN), NAN, NAN}),
-    _home_position_mutex(),
-    _home_position(Telemetry::Position {double(NAN), double(NAN), NAN, NAN}),
-    _in_air(false),
-    _armed(false),
-    _attitude_quaternion_mutex(),
-    _attitude_quaternion(Telemetry::Quaternion {NAN, NAN, NAN, NAN}),
-    _camera_attitude_euler_angle_mutex(),
-    _camera_attitude_euler_angle(Telemetry::EulerAngle {NAN, NAN, NAN}),
-    _ground_speed_ned_mutex(),
-    _ground_speed_ned(Telemetry::GroundSpeedNED {NAN, NAN, NAN}),
-    _gps_info_mutex(),
-    _gps_info(Telemetry::GPSInfo {0, 0}),
-    _battery_mutex(),
-    _battery(Telemetry::Battery {NAN, NAN}),
-    _flight_mode_mutex(),
-    _flight_mode(Telemetry::FlightMode::UNKNOWN),
-    _health_mutex(),
-    _health(Telemetry::Health {false, false, false, false, false, false, false}),
-    _rc_status_mutex(),
-    _rc_status(Telemetry::RCStatus {false, false, 0.0f}),
-    _position_subscription(nullptr),
-    _home_position_subscription(nullptr),
-    _in_air_subscription(nullptr),
-    _armed_subscription(nullptr),
-    _attitude_quaternion_subscription(nullptr),
-    _attitude_euler_angle_subscription(nullptr),
-    _camera_attitude_quaternion_subscription(nullptr),
-    _camera_attitude_euler_angle_subscription(nullptr),
-    _ground_speed_ned_subscription(nullptr),
-    _gps_info_subscription(nullptr),
-    _battery_subscription(nullptr),
-    _flight_mode_subscription(nullptr),
-    _health_subscription(nullptr),
-    _health_all_ok_subscription(nullptr),
-    _rc_status_subscription(nullptr),
-    _ground_speed_ned_rate_hz(0.0),
-    _position_rate_hz(-1.0)
+TelemetryImpl::TelemetryImpl(System &system)
+    : PluginImplBase(system), _position_mutex(),
```

Comment:
At some places like these (constructor member initialization), I feel old style is better readable. But, thats unavoidable I think.

Reply:
Could call, I was going to look at the changes and figure out how to prevent some of them.